docs/public-cloud/gitlab-setup.md

```diff
@@ -0,0 +1,225 @@
+# Gitlab Setup Guide for CAF Sample App
+
+This document outlines the steps necessary to set up a sample CAF (Cloud Adoption Framework) application in GitLab. 
+
+To summarize:
+
+0. Clone the [symphony](https://github.com/aztfmod/symphon) repo.
+  - For now use the following branch: [/davesee/bump_versions](https://github.com/aztfmod/symphony/tree/davesee/bump_versions)
+1. Set up the GitLab server with ```gitlab-server-setup.sh```
+2. Set up SSH Keys and clone repostories using ```clone-repos.sh```
+3. Deploy the launchpad landing zone manually to set up MSIs, using ```rover```
+4. Set up the Gitlab runners with ```install.sh```
+5. Run pipelines in Gitlab and troubleshoot as necessary
+
+# 1. GitLab Server Ssetup
+
+Set up the GitLab server using the instructions provided at the following location, specifically the section "Run the GitLab server deployment":
+
+* <a href="../../scripts/gitlab-server/README.md">/scripts/gitlab-server/README.md</a>
+
+# 2. Clone Repositories
```

Comment:
the second step should be launchpad deployment so the MSIs are created, they get added to GitLab in the clone step 3.

Reply:
Good catch. Swapped Steps 2 and 3 in both the summary at the top, and the content below.